CPS-XXXX/README.md

```diff
@@ -0,0 +1,82 @@
+---
+CPS: ?
+Title: Ouroboros not accounting for typical global network delays causes centralisation
+Status: Open
+Category: ?
+Authors: Terminada
+Proposed Solutions: []
+Discussions: https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-10-22
+License: CC-BY-4.0
+---
+
+# CPS-XXXX: Ouroboros not accounting for typical global network delays causes centralisation
+
```

Comment:
```suggestion
```
Titles for these documents are always generated from the tabular data in front & aren't part of the document itself.